src/poprox_concepts/domain/profile.py

```diff
@@ -15,3 +15,4 @@ class InterestProfile(BaseModel):
     click_locality_counts: dict[str, int] | None = None
     article_feedbacks: dict[UUID, bool] | None = None
     onboarding_topics: list[AccountInterest]
+    onboarding_entities: list[AccountInterest] = []  # Entity preferences (person, organisation, place)
```

Comment:
Since the type of these two properties is the same, I wonder if we really need two properties here. I'm not sure what to call a combined version though. `onboarding_selections`?

Reply:
@kluver What would you think about making this a dictionary instead of a list?

Reply:
Yeah, I wonder if we actually need an `AccountPreferences` object or something to wrap `AccountInterest` and provide some utility properties for accessing them by type etc